apps/src/templates/teacherNavigation/TeacherNavigationSidebarOption.tsx

```diff
@@ -0,0 +1,30 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import FontAwesomeV6Icon from '@cdo/apps/componentLibrary/fontAwesomeV6Icon/FontAwesomeV6Icon';
+import {BodyTwoText} from '@cdo/apps/componentLibrary/typography';
+
+import styles from './teacher-navigation.module.scss';
+
+interface TeacherNavigationSidebarOptionProps {
+  icon: string;
+  optionTitle: string;
+  isSelected: boolean;
+}
+
+const TeacherNavigationSidebarOption: React.FC<
+  TeacherNavigationSidebarOptionProps
+> = ({icon, optionTitle, isSelected}) => {
+  return (
+    <div
```

Comment:
We will most likely want to make this div into a link (probably a [NavLink](https://reactrouter.com/en/main/components/nav-link))

Not necessary for now, but you may want to look at what the NavLink can give you to get the active tab lined up correctly.

Reply:
Thanks for the tip!